lib/lita/handlers/jenkins.rb

```diff
@@ -77,11 +77,9 @@ def jenkins_list(response)
       end
 
       def headers
-        headers = {}
-        if auth = config.auth
-          headers["Authorization"] = "Basic #{Base64.encode64(auth).chomp}"
+        {}.tap do |headers|
+          headers["Authorization"] = "Basic #{Base64.encode64(config.auth).chomp}" if config.auth
```

Comment:
That's a good point - my personal "rule" is to allow a single line up to about 100-120 chars - my smaller screens and font sizes allow for that much in-screen.